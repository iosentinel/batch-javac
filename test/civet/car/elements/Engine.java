package car.elements;

import car.CarElementVisitor;

public class Engine implements CarElement {
  public void accept(CarElementVisitor visitor) {
    visitor.visit(this);
  }
}