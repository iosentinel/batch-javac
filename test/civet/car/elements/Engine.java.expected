package car.elements;

import car.CarElementVisitor;

public class Engine implements CarElement {
    
    public Engine() {
        super();
    }
    
    public void accept(CarElementVisitor visitor) {
        visitor.visit(this);
    }
    
    public static void accept$1002(car.CarElementVisitor visitor) {
        visitor.visit$1003();
    }
    
    public void accept$1012() {
        car.CarElementDoVisitor.visit$1014(this);
    }
}